zerver/worker/queue_processors.py

```diff
@@ -373,7 +374,11 @@ class MirrorWorker(QueueProcessingWorker):
     # management command, not here.
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-        mirror_email(email.message_from_string(event["message"]),
+        message = event["message"]
+        if not isinstance(event['message'], str):
+            message = event['message'].encode('utf8')
```

Comment:
@umairwaheed I think we can use `zerver.lib.str_utils.force_str` here instead; I'll just switch that when merging.